friendship/models.py

```diff
@@ -337,18 +340,6 @@ def add_friend(self, from_user, to_user, message=None):
 
         return request
 
-    def can_request_send(self, from_user, to_user):
```

Comment:
I'm not sure I'd remove this method. One hand, it's not documented in the README, so removing it wouldn't break compatibility. OTOH, I can see its usefulness and maybe it should be documented

Reply:
I think the name of this method implies that the user can send a request (if it returns true). But it returns the opposite - if a request was already sent. Therefore, if you decide to keep it, I would consider changing its name.

Reply:
+1 to changing the name, since it's undocumented anyway.

Reply:
Also, if you keep this method, maybe it's better to check both directions of the request - from user A to B, and from B to A.